ext/src/CoupledSDEs.jl

```diff
@@ -130,9 +133,11 @@ deterministic part of `ds`.
 """
 function DynamicalSystemsBase.CoupledODEs(
     sys::CoupledSDEs; diffeq=DEFAULT_DIFFEQ, t0=0.0)
+    prob = referrenced_sciml_prob(sys)
```

Comment:
This will be `nothing` if the system is not constructed through MTK. Doesn't this break normal systems? You cna't do `prob.f` then.